### PR TITLE
 Fix: Inconsistency in Title during creation of Resouce Request #9440

### LINKED ISSUE #9440: Inconsistency in title during creation of Resouce Request
**Describe the bug**
During creation of resource request, there is a text area provided for the 'Reason of Request'. But it is has given the title 'Description of Request'.
During creation of resource request and updation of resource request, it has title 'Description of Request' while after creation, it has title 'Reason'. This is creating inconsistency and confusion.

**To Reproduce**

1. Go to facilities tab.
2. Click on any facility
3. Click on Manage Facility and then in Resource Request
4. See the title 'Description of Resource'
5. Create it and then see it is replaced by 'Reason'
**Expected behavior**
The title during creation should be 'Reason of Resource' instead of 'Description of Resource'.

**Screenshots**
Image
Image

**Desktop:**

- OS: MacOS
- Browser Chrome

### PR DESCRIPTION
**Pull request by srijantrpth**
## Proposed Changes

- Fixes #9440 
- Changed the title of **Description of Request** to **Reason of Request** while creating new resource request and updating a resource request

Before: 

After: 

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Summary by CodeRabbit

- **New Features**
	- Updated terminology in the user interface, changing "Description of Request" to "Reason of Request" across multiple components and localization files.

- **Bug Fixes**
	- Ensured that the updated labels maintain clarity and consistency in user-facing messages.

- **Documentation**
	- Revised localization strings in English, Hindi, Kannada, Malayalam, Tamil, and other languages to enhance user understanding of input fields.

### CODE REVIEW

File: src/components/Resource/ResourceDetails.tsx [hunk 1]
```diff
@@ -127,7 +127,7 @@ export default function ResourceDetails(props: { id: string }) {
             </div>
             <div className="mt-1">
               <span className="font-semibold leading-relaxed">
-                Description of Request:{" "}
+                Reason of Request{" "}
```
[the remark this answers is not in the document]
Comment (srijantrpth) [reply src/components/Resource/ResourceDetails.tsx:130]: Fixed the consistency issue!

File: public/locale/hi.json [hunk 2]
```diff
@@ -643,8 +643,8 @@
   "req_atleast_one_lowercase": "कम से कम एक लोअर केस अक्षर की आवश्यकता है",
   "req_atleast_one_symbol": "कम से कम एक प्रतीक की आवश्यकता है",
   "req_atleast_one_uppercase": "कम से कम एक अपर केस की आवश्यकता है",
-  "request_description": "अनुरोध का विवरण",
-  "request_description_placeholder": "अपना विवरण यहाँ लिखें",
+  "request_reason": "अनुरोध का विवरण",
+  "request_reason_placeholder": "अपना विवरण यहाँ लिखें",
```
[the remark this answers is not in the document]
Comment (srijantrpth) [reply public/locale/hi.json:647]: Fixed the Translation Issue!

**Comment by srijantrpth** [comment]
@coderabbitai review

**Comment by srijantrpth** [comment]
Hi @ohcnetwork/care-fe-code-reviewers, Kindly review the PR and let me know if any changes are required. Thanks!

**Comment by srijantrpth** [comment]
Hi @rithviknishad @shivankacker  Kindly review the PR and let me know if any changes are required. Thanks!

**Review by rithviknishad — CHANGES_REQUESTED** [review]
https://github.com/ohcnetwork/care_fe?tab=readme-ov-file#translations

avoid adding other languages

**Comment by srijantrpth** [comment]
Hi @rithviknishad , I have reverted the translation for other languages. The JSON keys are same as the English language to ensure the consistency throughout the codebase

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**